docs/content/about/motivation.md

```diff
@@ -0,0 +1,17 @@
+---
+weight: 2
+---
+
+Mirantis and Docker have agreed to partner to maintain the shim code standalone outside Kubernetes, as a conformant CRI
+interface for the Docker Engine API. For Mirantis customers, that means that Docker Engine’s commercially supported
+version, Mirantis Container Runtime (MCR), will be CRI compliant. This means that you can continue to build Kubernetes
+based on the Docker Engine as before, just switching from the built in dockershim to the external one.
+
+Mirantis and Docker intend to work together on making sure it continues to work as well as before and that it
```

Comment:
What about moving this before "For Mirantis customers ..." so that it maintains flow with the partnership between Mirantis and Docker? 
Then we can highlight the (non) changes to Mirantis customers in the next paragraph